cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -59,10 +64,18 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.SortedMap;
 import java.util.TreeMap;
 import java.util.function.BiFunction;
 
+import static io.strimzi.operator.cluster.model.KafkaCluster.ANNO_STRIMZI_IO_FROM_VERSION;
+import static io.strimzi.operator.cluster.model.KafkaCluster.ANNO_STRIMZI_IO_KAFKA_VERSION;
+import static io.strimzi.operator.cluster.model.KafkaCluster.ANNO_STRIMZI_IO_TO_VERSION;
+import static io.strimzi.operator.cluster.model.KafkaCluster.ENV_VAR_KAFKA_CONFIGURATION;
+import static io.strimzi.operator.cluster.model.KafkaConfiguration.INTERBROKER_PROTOCOL_VERSION;
+import static io.strimzi.operator.cluster.model.KafkaConfiguration.LOG_MESSAGE_FORMAT_VERSION;
```

Comment:
I agree we need to be consistent. I've made the case before for common static methods to be imported (e.g. `Arrays.asList()`, `Collections.emptyMap()`). I agree when they're not methods we regularly use in this code base then explicit qualification is preferred.

In this case I feel that when _I_ read the code in `kafkaUpgradePhase1()`, for example, that qualifying the constants adds nothing to understanding the code. When reading that code I don't care where the constant is defined (or even what it's value is, because I can guess well enough from the constant name). I care about the algorithm: what gets updated, and in what order. TBH I rarely care where a constant comes from. I _sometimes_ care what it's value is, but that's not what you're complaining about.

So I'm not keen on changing this _particular_ code, because I think it would make the code less clear.

That said, if others disagree I'll happily change it, but I think the decision should be based on how easy it is to read the code, not simply on a personal preference one way or the other.